src/tests/ftest/cat_recovery/ddb.py

```diff
@@ -0,0 +1,666 @@
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+import ctypes
+import re
+from ClusterShell.NodeSet import NodeSet
+
+from apricot import TestWithServers
+from pydaos.raw import IORequest, DaosObjClass
+from general_utils import create_string_buffer, report_errors, run_pcmd, \
+    distribute_files, DaosTestError, get_clush_command, run_command
+from ddb_utils import DdbCommand
+from exception_utils import CommandFailure
+
+
+SAMPLE_DKEY = "Sample dkey"
+SAMPLE_AKEY = "Sample akey"
+SAMPLE_DATA = "Sample data"
+
+
+class DdbTest(TestWithServers):
+    """Test ddb subcommands.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DdbTest object."""
+        super().__init__(*args, **kwargs)
+        self.ioreqs = []
+        self.dkeys = []
+        self.akeys = []
+        self.data_list = []
+
+    def insert_objects(self, object_count, dkey_count, akey_count):
+        """Insert objects, dkeys, akeys, and data into the container.
+
+        Inserted objects: self.ioreqs
```

Comment:
We should fix this spelling error. Maybe by adding an entry to words.dict

Reply:
Done